2024-10-05 18:42:08,009 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:259]
2024-10-05 18:42:09,161 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:259]
2024-10-05 18:42:13,909 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:259]
2024-10-05 18:45:07,370 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:259]
2024-10-05 18:45:08,135 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:259]
2024-10-05 18:45:27,132 ERROR: Unhandled exception: Multiple classes found for path "Cliente" in the registry of this declarative base. Please use a fully module-qualified path. [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:308]
2024-10-05 18:49:56,730 ERROR: Unhandled exception: One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper[Factura(factura)]'. Original exception was: Multiple classes found for path "Cliente" in the registry of this declarative base. Please use a fully module-qualified path. [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:308]
2024-10-05 19:22:02,475 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 19:22:03,555 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 19:22:06,900 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 19:23:27,436 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 19:23:28,358 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 19:23:47,843 ERROR: Error rendering facturacion template: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:27]
2024-10-05 19:23:47,843 ERROR: Unhandled exception: error.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 19:25:27,465 ERROR: Error rendering facturacion template: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:27]
2024-10-05 19:25:27,466 ERROR: Unhandled exception: error.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:06:05,959 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:19:37,507 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:20:31,809 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:21:42,733 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:48:28,496 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:49:02,354 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:51:30,277 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:51:43,750 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:51:48,035 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:51:50,183 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:51:57,817 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:52:40,205 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:52:44,864 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:53:21,612 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:53:40,710 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:53:42,399 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.facturas'. Did you mean 'facturacion.pre_facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:53:50,260 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:54:14,495 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:54:16,770 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:54:37,935 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:55:19,353 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 20:55:35,708 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:55:39,901 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:55:41,564 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:55:42,976 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:58:00,421 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:58:01,894 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 20:58:06,468 ERROR: Unhandled exception: Could not build url for endpoint 'facturacion.index'. Did you mean 'facturacion.facturas' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:00:03,797 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:06:27,898 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:06:58,051 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:07:06,587 ERROR: Unhandled exception: Could not build url for endpoint 'redirect_to_facturas'. Did you mean 'redirect_to_bancos' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:07:14,537 ERROR: Unhandled exception: Could not build url for endpoint 'redirect_to_facturas'. Did you mean 'redirect_to_bancos' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:07:20,047 ERROR: Unhandled exception: Could not build url for endpoint 'redirect_to_facturas'. Did you mean 'redirect_to_bancos' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:07:26,749 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:07:45,966 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:07:48,567 ERROR: Unhandled exception: Could not build url for endpoint 'redirect_to_facturas'. Did you mean 'redirect_to_bancos' instead? [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:07:53,466 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:11:39,416 ERROR: Unhandled exception: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:11:43,331 ERROR: Unhandled exception: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:11:46,837 ERROR: Unhandled exception: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
2024-10-05 21:54:14,626 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 21:58:14,562 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 22:00:05,602 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:260]
2024-10-05 22:01:05,552 ERROR: Unhandled exception: facturacion/base_facturacion.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:309]
